### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by mariem460**

### CODE REVIEW

File: javascript/javascript2/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+let newNumbers = [];
+
+const doubledNumbers = numbers.filter((number)=>{
+    if(number%2==0){
+        console.log(`even numbers are: ${number}`)
+    }else if(number%2!==0){
+        let oddNumbers = number*2;
+        newNumbers.push(oddNumbers);
+    }
+});
```
Comment (dinythomas89) [inline javascript/javascript2/week2/index.js:12]: you can use the chaining property here using filter and map to print the odd numbers as doubled

File: javascript/javascript2/week2/movies.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    {"title": "A Dirty Shame","year": 2004,"rating": 5.1,"votes": 11191,"running_times": 5340},
+    {"title": "A Dog's Purpose","year": 2017,"rating": 6.9,"votes": 38675,"running_times": 6000},
+    {"title": "A Face in the Crowd","year": 1957,"rating": 8.2,"votes": 10910,"running_times": 7560},
+    {"title": "hello world hello","year": 1957,"rating": 8.2,"votes": 10910,"running_times": 7560},
+    {"title": "A Fantastic Fear of Everything","year": 2012,"rating": 5.9,"votes": 12446,"running_times": 6000}
+];
+//short and log titles
+const shortTiteles =  movies.filter(movie=> movie.title.length<10).map(movie=>movie.title);
+moviesShortTitles.push(shortTiteles)
+console.log(moviesShortTitles);
```
Comment (dinythomas89) [inline javascript/javascript2/week2/movies.js:133]: You don't need to create another array, as the filter function itself will generate an array. That means in here shortTiteles is an array created after filtering the movies. So no need for 'moviesShortTitles'

File: javascript/javascript2/week2/movies.js [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+const longTiteles =  movies.filter(movie=> movie.title.length>10).map(movie=>movie.title);
+longMovieTitle.push(longTiteles)
```
Comment (dinythomas89) [inline javascript/javascript2/week2/movies.js:135]: Same as above one

**Review by dinythomas89 — APPROVED** [review]

File: javascript/javascript2/week2/movies.js (hunk 2 above)
Comment (mariem460) [reply javascript/javascript2/week2/movies.js:133]: yes make sense  :)

**Pull request merged**